internal/statemachine/state_machine_test.go

```diff
@@ -790,3 +790,16 @@ func TestFailedPostProcessGadgetYaml(t *testing.T) {
 		asserter.AssertErrContains(err, "disallowed for security purposes")
 	})
 }
+
+func TestPartialImage(t *testing.T) {
```

Comment:
nitpicking here, but this is not using new `partial` feature, but `min-size`, so the test name should be different